### ISSUE TITLE
Focus changes to Play+Skip in Recording Details screen after user has already switched focus to another button

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/mythbox#108]
```
What steps will reproduce the problem?
1. Go to the recordings details screen for a recording with comm breaks
2. Quickly change focus to 'Edit Schedule'
3.  After the 'Comm Breaks' field gets populated, focus changes to the 
'Play+Skip' button. This is annoying because it changes focus against out from 
underneath the user's selection.

What is the expected output? What do you see instead?
Expected: button focus should not change to 'Play+Skip' if the user has already 
changed the selection

```

Original issue reported on code.google.com by `semir.pa...@gmail.com` on 17 Jul 2010 at 7:25

**Comment by GoogleCodeExporter** [comment]
```
Fixed in changeset 72
```

Original comment by `semir.pa...@gmail.com` on 17 Jul 2010 at 7:32
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Lets see if revision links work --  fixed in revision 9a0486f5b0
```

Original comment by `semir.pa...@gmail.com` on 17 Jul 2010 at 7:35

**Comment by GoogleCodeExporter** [comment]
```
Lets see if friendly revision links work -- fixed in revision 72
```

Original comment by `semir.pa...@gmail.com` on 17 Jul 2010 at 7:36
